docs/traceflow-guide.md

```diff
@@ -0,0 +1,81 @@
+# Traceflow User Guide
+
+Antrea supports using Traceflow for network diagnosis.
+Creating a new Traceflow CRD triggers Traceflow module to inject packet into OVS, provide various observation points
+along the packet's path and populate these observations into status of Traceflow CRD.
+Users can start a new trace simply from either Kubectl or Antrea-Octant-Plugin and view Traceflow result via CRD or UI graph.
+We will also provide corresponding Antctl to start a new trace in the near future.
+
+For antrea-octant-plugin installation, please refer to [antrea-octant-installation](/docs/octant-plugin-installation.md).
+
+## Table of Contents
+
+- [Start a New Trace](#Start-a-New-Trace)
+  - [Using kubectl and YAML file](#using-kubectl-and-YAML-file)
+  - [Using Octant with antrea-octant-plugin](#Using-Octant-with-antrea-octant-plugin)
+- [View Traceflow Result and Graph](#View-Traceflow-Result-and-Graph)
+- [View Traceflow CRDs](#View-Traceflow-CRDs)
+
+## Start a New Trace
+You can choose to use either Kubectl together with YAML file or Octant UI to start a new trace.
+
+### Using kubectl and YAML file
```

Comment:
do you also think we should explain each supported field in detail, perhaps a little later in this document ? for example `destination` also accepts IPs instead of a Pod.. this does not explain me all the available features of traceflow..

Reply:
Thanks for the suggestion. Will add details for this part.

Reply:
We support Pod as source, Pod/Svc/IP as destination.

Reply:
UI now doesn't support destination to be IP as we did not know this previously. @ZhangYW18 We can enhance this part together with service support from UI side.